pkg/dom0-ztools/Dockerfile

```diff
@@ -58,3 +58,5 @@ RUN addgroup -S tpms
 RUN addgroup -S vtpm
 RUN adduser -S -D -H -h /nonexistent -s /bin/false -g "vtpm" -G vtpm vtpm
 RUN addgroup vtpm tpms
+
```

Comment:
would you please add comment about the group?

Reply:
now that #3989 is merged, please rebase and use a fixed id for the group.

Reply:
Yeah I'll add both requests